apps/rush-lib/src/logic/policy/GitEmailPolicy.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as colors from 'colors';
+import * as os from 'os';
+
+import { GitPolicy } from './GitPolicy';
+import { RushConfiguration } from '../../api/RushConfiguration';
+import { AlreadyReportedError } from '../../utilities/AlreadyReportedError';
+import { Utilities } from '../../utilities/Utilities';
+
+interface IResultOrError<TResult> {
+  error?: Error;
+  result?: TResult;
+}
+
+export class GitEmailPolicy extends GitPolicy {
+  public static getUserEmail(rushConfiguration: RushConfiguration): string {
+    // Determine the user's account
+    // Ex: "bob@example.com"
+    const emailResult: IResultOrError<string> = GitEmailPolicy.tryGetEmail();
+    if (emailResult.error) {
+      console.log(
+        [
+          `Error: ${emailResult.error.message}`,
+          'Unable to determine your Git configuration using this command:',
+          '',
+          '    git config user.email',
+          '',
+          `If you didn't configure your e-mail yet, try something like this:`,
+          ...GitEmailPolicy.getEmailExampleLines(rushConfiguration)
+        ].join(os.EOL)
+      );
+      throw new AlreadyReportedError();
+    }
+
+    return emailResult.result || '';
+  }
+
+  private static tryGetEmail(): IResultOrError<string> {
```

Comment:
>tryGetEmail [](start = 17, length = 11)

Since Git calls this `user.email`, we could call our variable `userEmail`.  We could also call it `gitEmail`.  But `email` by itself is a little ambiguous and difficult to search for in the code base.